utils.go

```diff
@@ -55,10 +62,9 @@ const (
 )
 
 func generateAlias(n int) string {
```

Comment:
as I can see we have utils.go and test_utils.go
is no it that any random generated data stuff must be in the test section?

Reply:
Yes, these are still some leftovers from "old" code that used that. But yes, you're correct, this should be moved, but I wanted to limit the scope of this PR a bit as it already started to become a huge "refactor ALL the code"-PR.